core/core_integrations/slack.js

```diff
@@ -605,8 +584,9 @@ exports.stop = function() {
 			while (numSocketsToShutDown > 0 && !clearedMessageQueue && !clearedTimeouts) {
 				sleep.usleep(500000);
```

Comment:
Given this is slack related, do you mind fixing the `npm install sleep` problems on Windows while we are here.